src/style.css

```
@@ -1,3 +1,76 @@
+:root {
+  --cyan: #00acdc
```

use more meaningful names
```suggestion
  --accent-color: #00acdc
```